src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -743,6 +750,7 @@ func (c *controller) UploadLogs(ctx context.Context, cancellog context.CancelFun
 					return err == nil
 				})
 			}
+			_ = c.ic.ClusterLogProgressReport(progress_ctx, c.ClusterID, models.LogsStateCompleted)
```

Comment:
so we can stuck with waiting for logs if this api call will fail?

Reply:
There is a timeout for that, so eventually we will stop waiting